src/linked_list.js

```diff
@@ -0,0 +1,172 @@
+'use strict'
+
+class Node {
+  constructor(data, next) {
+    this.data = data
+    this.next = next
+  }
+}
+
+export default class LinkedList {
+  constructor() {
+    this.list = null
```

Comment:
I think this could be more clear. I know this refers to the head node because of your other functions but it would be more clear if this was named this.head or something similar, as you are not referring to the entire list but a singular node in that list.